warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/ShardedTableMapFile.java

```diff
@@ -322,33 +317,30 @@ public static Path createShardedMapFile(Logger log, Configuration conf, Path wor
      *
      * @param log
      *            logger for reporting errors
-     * @param instance
-     *            Accumulo instance to query shard locations
-     * @param credentials
-     *            Accumulo access credentials for querying shard locations
+     * @param accumuloHelper
+     *            Accumulo helper to query shard locations
      * @param shardedTableName
      *            name of the shard table--the table whose locations we are querying
      * @return a map of KeyExtent to the location of those key extents in accumulo
```

Comment:
The @return description is no longer accurate.

Reply:
Fixed in 669fc27